build-system/compile/compile.js

```diff
@@ -116,6 +115,19 @@ function compile(
     );
   }
 
+  function getSourceMapBase() {
+    if (!!argv.type) {
+      // Production build, point sourcemap to fetch files from correct GitHub tag.
+      return `https://raw.githubusercontent.com/ampproject/amphtml/${internalRuntimeVersion}/`;
+    } else if (argv.sourcemap_url) {
+      return String(argv.sourcemap_url).replace(
+        '{version}',
```

Comment:
@mdmower I assume this is the only place where the version passed in via the command line flag is substituted, correct?

Regarding the actual substitution string, we already use `$internalRuntimeVersion$` in our source code:

https://github.com/ampproject/amphtml/blob/bb9369319000b84a5a77ca24157762ff70a68a2e/src/internal-version.js#L17-L28

Would it work if you used the same string for URL substitution as well?

```
gulp dist --sourcemap_url "https://example.com/$internalRuntimeVersion$/root/" --version_override "12345"
```

Reply:
>  I assume this is the only place where the version passed in via the command line flag is substituted, correct?

Correct.

> we already use $internalRuntimeVersion$... Would it work if you used the same string for URL substitution as well?

Thanks for pointing that out! Sure, I'll make the change.

Reply:
@rsimha Actually, I'm questioning whether string `$internalRuntimeVersion$` is the best choice for a user-facing flag. I've tried to make a consistent distinction between a version and a runtime version, reserving the latter to mean (config + version).

Reply:
Gotcha. I have no strong personal preference here, since it is a limited use case that you are more familiar with. (Perhaps `$version$` is a good compromise?)

Once decided, we should mention the actual substitution string in the flag notes for `--sourcemap_url`.

Reply:
I opted to keep `{version}` since shells would treat `$version$` like an environment variable `$version` with an appended `$` in common use cases like
```
gulp dist --sourcemap_url "https://example.com/$version$/"
```